examples/mathml/Main.hs

```diff
@@ -22,11 +23,38 @@ updateModel _ = pure ()
 viewModel :: Model -> View ()
 viewModel _ =
     math_
-        []
-        [ nodeMathml
-            "msup"
-            []
-            [ nodeMathml "mi" [] [text "x"]
-            , nodeMathml "mn" [] [text "2"]
+        [display_ "block"]
```

Comment:
Changed this to something slightly more complex (quadratic formula).
Works fine for me in chrome and firefox
![Screenshot from 2025-05-11 15-32-49](https://github.com/user-attachments/assets/e05acdb3-b0e4-45fa-86cb-898018c10efc)

Reply:
hell yes